django/thunderstore/community/models/community_site.py

```diff
@@ -29,6 +32,7 @@ class CommunitySite(TimestampMixin, models.Model):
         on_delete=models.CASCADE,
     )
     is_listed = models.BooleanField(default=True)
+    is_pre_subdomain_change_site = models.BooleanField(default=False)
```

Comment:
Do we really need this? I realize this was probably based on my request, but having thought about it a bit further, I'd imagine we drop the entire `CommunitySite` concept for communities that don't have a subdomain, meaning this wouldn't have to be tracked here.

Reply:
Hmm, seems like we don't need this. Looks like after all I'm not even using this variable anywhere to check anything. I just need to rework the commits to not make irrelevant migrations.